models/SentryAppender.cfc

```diff
@@ -42,7 +42,10 @@ component extends="coldbox.system.logging.AbstractAppender" accessors=true{
 		}
 	
 		// Is this an exception or not?
-		if( isStruct( extraInfo ) && extraInfo.keyExists( "StackTrace" ) && extraInfo.keyExists( "message" ) && extraInfo.keyExists( "detail" ) ){
+		if( 
+			( isStruct( extraInfo ) || isObject( extraInfo ) )
+            && extraInfo.keyExists( "message" ) && extraInfo.keyExists( "detail" ) 
```

Comment:
@jclausen You removed the `extraInfo.keyExists( "StackTrace" )` check.  Did you run into a scenario where that key didn't exist?  I had that check in place since it seemed likey that a random struct may have a `message` and `detail` key while not being a cfcatch struct, but it would be more unlikely to have those two along with a `stacktrace` key.